panda.py

```diff
@@ -0,0 +1,224 @@
+from pxr import Usd, Vt
+import json
```

Comment:
It is good form to organize imports by:
1. System
2. Specialty
3. Alphabetcal order

So as a result:
```python
# System
import json

# USD
from pxr import Usd, Vt
```